gatsby/src/pages/drugs/{BnfDrug.slug}/medicinal-forms.tsx

```diff
@@ -1,28 +1,62 @@
-import { graphql } from "gatsby";
-import { FC } from "react";
+import { graphql, Link } from "gatsby";
+import { FC, ReactElement } from "react";
 import striptags from "striptags";
+import { type Merge } from "type-fest";
 
 import {
 	type FeedMedicinalForms,
 	type FeedMedicinalForm,
+	FeedLabel,
 } from "@nice-digital/gatsby-source-bnf";
 
+import { Accordion, AccordionTheme } from "@/components/Accordion/Accordion";
+import labelStyles from "@/components/CautionaryAndAdvisoryLabel/CautionaryAndAdvisoryLabel.module.scss";
 import { DetailsPageLayout } from "@/components/DetailsPageLayout/DetailsPageLayout";
 import { Prep } from "@/components/Prep/Prep";
-import { type QueryResult, type WithSlugDeep } from "@/utils";
+import { type QueryResult, WithSlug } from "@/utils";
+
+import styles from "./medicinal-forms.module.scss";
+
+type queryMedicinalForm = Merge<
+	WithSlug<FeedMedicinalForm>,
+	{
+		cautionaryAndAdvisoryLabels?: {
+			label: FeedLabel;
```

Comment:
I think we need to `Exclude` the `description` property here now we're not selecting it in the query

Reply:
Unresolving this as we're still using `FeedLabel` here as the type, which includes `description` which we're not querying for. We're only querying:
![image](https://user-images.githubusercontent.com/893676/160688465-d9a31aa8-cb20-421c-b485-b276042a2874.png)